lib/web_console/fake_middleware.rb

```diff
@@ -0,0 +1,50 @@
+require "action_view"
+require "action_dispatch"
+require "web_console"
+
+module WebConsole
+  class FakeMiddleware
```

Comment:
Can this live in `lib/web_console/testing/fake_middleware.rb`? This is to follow the example of other Rails component grouping their testing utilities in similar module.